docs-v2/integration-templates/overview.mdx

```diff
@@ -101,6 +101,10 @@ As Nango and its community expand, we're looking forward to offering hundreds of
     Sync Zendesk tickets and articles
   </Card>
 
+  <Card title="Lever" href="/integration-templates/lever" icon="briefcase">
+    Sync Lever opportunities, applications, feedback, interviews, notes and offers for a specific opportunity. Postings and questions for a specific job posting. Actions to create; a note for a specific opportunity and an opportunity.
+  </Card>
+
```

Comment:
I think there is a missing word at the end `and an opportunity`. also extra `;` after `create`